index.js

```diff
@@ -22,7 +26,9 @@ function getPercentageScore(score) {}
  * @param {object} score2 - An object representing another score of an assignment. Has the same shape as `score1`.
  * @returns {number} A rounded integer, representing the highest percentage scored among the two given scores.
  */
-function getHighestOfTwoScores(score1, score2) {}
+function getHighestOfTwoScores(score1, score2) {
+  return Math.max(getPercentageScore(score1), getPercentageScore(score2))
```

Comment:
Nicely concise code on all three solutions, and great use of the earlier function here!